website/docs/table-design/table-types/pk-table/merge-engines/default.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_label: Default
```

Comment:
Maybe `LatestRow`

Reply:
I recommend to use `Default` or `Default (LastRow)`